src/app/test/test.component.ts

```diff
@@ -155,11 +152,6 @@ export class TestComponent implements OnInit {
         next: (response: TestResult): void => {
           report.reranReport = this.createReranReport(response);
         },
-        error: () =>
-          catchError((error: HttpErrorResponse): Observable<any> => {
-            report.error = error.message;
-            return of(error);
-          }),
```

Comment:
This code can't be replaced for the handleError method, so can  you keep this code here?

Reply:
I added this back